website/docs/foundations/typography/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,195 @@
+<Hds::Text::Body @size="300" @weight="medium">Body 300 (16px) - Medium</Hds::Text::Body>
+<br/>
+<Hds::Text::Body @size="300" @weight="regular">Body 300 (16px) - Regular</Hds::Text::Body>
+<br/>
+<br/>
+<Hds::Text::Body @size="200" @weight="semibold">Body 200 (14px) - Semibold</Hds::Text::Body>
+<br/>
+<Hds::Text::Body @size="200" @weight="medium">Body 200 (14px) - Medium</Hds::Text::Body>
+<br/>
+<Hds::Text::Body @size="200" @weight="regular">Body 200 (14px) - Regular</Hds::Text::Body>
+<br/>
+<br/>
+<Hds::Text::Body @size="100" @weight="semibold">Body 100 (13px) - Semibold</Hds::Text::Body>
+<br/>
+<Hds::Text::Body @size="100" @weight="medium">Body 100 (13px) - Medium</Hds::Text::Body>
+<br/>
+<Hds::Text::Body @size="100" @weight="regular">Body 100 (13px) - Regular</Hds::Text::Body>
+
+### Code
+
+Code styles are used exclusively for text representing code in the UI (like the [Code Block component](/components/code-block)). 
+
+Code text styles come in the following weights and sizes:
+
+<Hds::Text::Code @size="300" @weight="bold">Code 300 (16px) - Bold</Hds::Text::Code>
+<br/>
+<Hds::Text::Code @size="300" @weight="regular">Code 300 (16px) - Regular</Hds::Text::Code>
+<br/>
+<br/>
+<Hds::Text::Code @size="200" @weight="bold">Code 200 (14px) - Bold</Hds::Text::Code>
+<br/>
+<Hds::Text::Code @size="200" @weight="regular">Code 200 (14px) - Regular</Hds::Text::Code>
+<br/>
+<br/>
+<Hds::Text::Code @size="100" @weight="bold">Code 100 (13px) - Bold</Hds::Text::Code>
+<br/>
+<Hds::Text::Code @size="100" @weight="regular">Code 100 (13px) - Regular</Hds::Text::Code>
+
+## How typography is used
+
+### Body content
+
+Body 200 is the default font size for body content. Scaling up or down is used as a means to create visual hierarchy in page UI, or component usage. As an example, the Button, `medium` (default) variant is sized at Body 200 while the `large` variant is Body 300.
+
+![Two buttons, one medium and one large](/assets/foundations/typography/typography-button-example.png)
+
+Similarly, form fields' labels and input texts are Body 200, while any other secondary information, like helper text or error messages are sized at Body 100. This ensures that primary information is more prominent with the larger size for visual hierarchy.
+
+![A form field with label, helper text, an input field in an error state, and an error message.](/assets/foundations/typography/typography-form-example.png)
+
+For more details about form patterns, read the [form patterns](/patterns/form-patterns) documentation.
+
+### Semantic heading or not?
+
+The [Text](/components/text) component does not equate a specific style (Body, Display, or Code) with a specific HTML semantic structure (like `<p/>`, `<h1/>` or `<span>`), it is up to the consumer to define the logical semantic element for the each piece of text. To ensure accessible page content, the scaling of the Display styles often are associated with the scaling of the HTML headings such as:
+
+- Display 500 -> H1
```

Comment:
Writing an arrow this way doesn't get announced as you might expect with a screen reader. VoiceOver ignores the "-" and just says "Display 500 greater than H1". There is an ascii character for right arrow that announces correctly that you can use instead.

```suggestion
- Display 500 → H1
```

Reply:
Oh wow, great to know, thanks for the education :D